QHyper/converter.py

```diff
@@ -34,7 +37,7 @@ def use_slacks(const: int | float, label: str) -> Polynomial:
             {(f"{label}_{i}",): v
              for i, v in enumerate(slack_coefficients)}
         )
-
+    
```

Comment:
https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

Reply:
I followed these suggestions, but nothing changed, whitespace remains when saving the file :--(